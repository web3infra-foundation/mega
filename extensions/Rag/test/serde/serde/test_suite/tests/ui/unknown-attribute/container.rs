use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(abc = "xyz")]
struct A {
    x: u32,
}

fn main() {}
