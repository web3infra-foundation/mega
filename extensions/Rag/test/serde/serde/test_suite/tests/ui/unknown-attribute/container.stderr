error: unknown serde container attribute `abc`
 --> tests/ui/unknown-attribute/container.rs:4:9
  |
4 | #[serde(abc = "xyz")]
  |         ^^^
